content/id/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -0,0 +1,226 @@
+---
+reviewers:
+- girikuncoro
+title: Kontainer Sementara
```

Comment:
how about adding the original "Ephemeral", since we are introducing "new" term in Bahasa Indonesia
```suggestion
title: Kontainer Sementara (Ephemeral)
```